components/accept-terms.jsx

```diff
@@ -6,11 +6,10 @@ const DEFAULT_AGE_RESTRICTION = '16';
 const DEFAULT_PRIVACY_POLICIES_POSITION = 'top';
 
 export function AcceptTerms({
-	isAuthFirstAccount = false,
-	isAuthFirstPayment = false,
+	isAccessPage = false,
+	isPaymentPage = false,
```

Comment:
The component should not be designed for a given consumer.
The AcceptTerms component should not reflect or rely on logic from next-subscribe.
AccessPage and PaymentPage are steps of the buy flow and have got nothing to do with the component.
What we do want are some option so that we can configure the component behaviour from the consumer.
So we need to identify what `isAccessPage` does in the component and reflect this as a feature.

If I'm not mistaken, `isAccessPage` causes privacy policy terms to be displayed.
`isPaymentPage` is used to drive is we display B2B terms.

So I would have the following flags:
```suggestion
	withPrivacyPolicyTerms = false,
	withB2bTerms = false,
```

We're now agnostic to our consumer and the steps of the buy flow are no longer reference in the component.